tests/vulture/run_vulture.sh

```diff
@@ -10,23 +10,13 @@ option_string # Used when creating custom actions for argparse
 # replace new line with commas and delete comments
 IGNORE_NAMES=$(echo "$IGNORE_NAMES" | tr '\n' ',' | sed 's/#.*//')
 
-find_scripts() {
```

Comment:
you mixed here the commits as well. This was introduced in the first commit, please fix it there or squash.